### ISSUE TITLE
"bosco run" fails to remove some already running containers

### ISSUE THREAD

**Issue by ksnll** [tes/bosco#247]
If you have some containers already running, bosco fails to stop them during "bosco run"
Solution for now is to run ```docker stop $(docker ps -q)``` before using ```bosco run```
@cliftonc 

```[10:06:09] Bosco: Did not start 6 services that were already running.  Use --verbose to see more detail.
[10:06:09] Bosco: Launching 10 service processes with parallel limit of 7 ...
[10:06:09] Bosco: There was an error running service-site-assets: Error: (HTTP code 409) unexpected - You cannot remove a running container 492c0dc8fbfcba7f222b1aec4fcd0046a79fd1f96cf27dd646489ae03674b8cf. Stop the container before attempting removal or force remove
[10:06:09] Bosco: There was an error running service-page-composer: Error: (HTTP code 409) unexpected - You cannot remove a running container d8024667b0909516bc7d966af88b3447f3f8d085f7926d574b662be8f32123c1. Stop the container before attempting removal or force remove
[10:06:09] Bosco: There was an error running service-layout-template: Error: (HTTP code 409) unexpected - You cannot remove a running container b7e35c7baa839cd662552f6e09b0b8f4766518fb809c56782a6f5452722c74b8. Stop the container before attempting removal or force remove

**Comment by ksnll** [comment]
https://github.com/tes/bosco/pull/248

**Comment by ksnll** [comment]
fixed in 0.5.116
